src/scrape_up/github/__init__.py

```diff
@@ -49,3 +49,17 @@ def get_bio(self):
         except:
             message = f"Bio not found for username {self.username}"
             return message
+        
+        
+    def get_repo(self):
+        """
```

Comment:
Hey @ayushanand308, include this method in the `endpoint.md` file

Reply:
Done.